include/seqan3/io/stream/parse_condition_detail.hpp

```diff
@@ -344,7 +344,7 @@ struct is_in_interval_type : public parse_condition_base<is_in_interval_type<int
     {
         data_t ret{};
 
-        for (uint8_t i = interval_first; i <= interval_last; ++i)
+        for (size_t i = interval_first; i <= static_cast<size_t>(interval_last); ++i)
```

Comment:
why this change?

Reply:
if you have full range (min to max) you overflow and never quit.